run_integration_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

I think you want this for compatibility across systems (I had to change it to this go get it to work for me and have seen similar issues in the past).

Reply:
Addressed in 143b1342a11bff1ffd6568b671e54c8a1b7b646f.